internals/cli/cmd_exec.go

```diff
@@ -223,6 +216,17 @@ func (cmd *cmdExec) Execute(args []string) error {
 	}
 }
 
+// useTerminal returns true if -t is given, or if stdoutIsTerminal is true.
+func (cmd *cmdExec) useTerminal(stdoutIsTerminal bool) bool {
```

Comment:
As a side note, this function is not looking so great.  The behavior is a bit confusing, the documentation is incomplete, and the fact it uses an argument that is irrelevant for the function logic is an anti-pattern (just use the flag at the call site instead).  All of this to use the values of two trivial boolean flags.

Reply:
Yeah, fair enough. This may be a moot point if we change the behaviour to use FD 2 anyway, but otherwise let's just duplicate this (simple) logic in both places (possibly with a short comment at both sites saying, "this duplicates the logic in Xyz").

Reply:
Removed it and duplicated the logic as Ben suggested.